### ISSUE TITLE
[Meds / Meds Management Improvements] Align page titles/headings/breadcrumbs/links

### ISSUE THREAD

**Issue by ckeif-iat** [department-of-veterans-affairs/va.gov-team#139756]
## Context
Hanging links were added in [#138106](https://github.com/department-of-veterans-affairs/va.gov-team/issues/138106). This is a finding from [the accessibility review](https://github.com/department-of-veterans-affairs/va.gov-team/issues/138106#issuecomment-4253322436).

<img width="1839" height="1266" alt="Image" src="https://github.com/user-attachments/assets/b4284763-1786-4a6b-ace4-c6f6b073bf24" />

## Description

The page title, heading, and breadcrumbs should be aligned for each tab, but they are not.

**Landing/refill**
- Title: Refill prescriptions - Medications | Veterans Affairs
- Heading: Medications
- Breadcrumbs: Medications
- Hanging link: Medication Refills

<img width="1839" height="1266" alt="Image" src="https://github.com/user-attachments/assets/ed649bf4-1677-4a7f-bdea-b7e640edef75" />

**List**
- Title: MHV Medications | Veterans Affairs
- Heading: Medications list
- Breadcrumbs: Medications list
- Hanging Link: Medications List

<img width="1839" height="1273" alt="Image" src="https://github.com/user-attachments/assets/ec34b39b-84bf-4ed1-9fbf-6ea6d396f161" />

**Refill Status**
- Title: MHV Medications | Veterans Affairs
- Heading: Prescription refill status
- Breadcrumbs: Refill Status
- Hanging Link: Refill Status

<img width="1841" height="1271" alt="Image" src="https://github.com/user-attachments/assets/fb6cd2cf-6f5e-4054-ae38-3330403fd39b" />

Additionally, the page title does not change when a link is clicked.

## Resources
- [MMI v2 designs](https://www.figma.com/design/VunyfNQ9x7Bt58D9B2qnW5/%F0%9F%92%A1-Working-file--MHV-Medications---Devices-?m=auto&node-id=10237-42881&t=68MSVBpbopu7q9NK-1)
- https://www.w3.org/WAI/WCAG22/Understanding/page-titled

## Tasks
- [ ] Make the title/heading/breadcrumbs/hanging link text consistent for each page.
- [ ] Page title changes appropriately when link is clicked

## Acceptance Criteria
- [ ] Make the title/heading/breadcrumbs/hanging link text consistent for each page.
- [ ] Page title changes appropriately when link is clicked
